code/modules/jobs/job_types/atmospheric_technician.dm

```diff
@@ -26,11 +26,14 @@
 
 	display_order = JOB_DISPLAY_ORDER_ATMOSPHERIC_TECHNICIAN
 	threat = 0.5
-	
-	family_heirlooms = list(
-		/obj/item/lighter,
-		/obj/item/lighter/greyscale,
-		/obj/item/storage/box/matches
+
+	family_heirlooms = list(					//LUMOS CHANGE - Heirloom touch up
+		//obj/item/lighter,
+		//obj/item/lighter/greyscale,
+		//obj/item/storage/box/matches
```

Comment:
I feel like a lighter heirloom is quite on theme for an atmos tech. Though I suppose if they want a lighter they can just spawn with one.

Reply:
They still get a lighter, the cool slime one.